mongo/job_store.go

```diff
@@ -239,6 +284,26 @@ func (m *JobStore) UpdateJob(updates bson.M, s *mgo.Session, id string) error {
 	return nil
 }
 
+// UpsertTask creates a new task document or overwrites an existing one
+func (m *JobStore) UpsertTask(jobID, taskName string, task models.Task) (err error) {
```

Comment:
Parentheses not required for methods/funcs signatures returning single value.

Reply:
good point - changed